### PR TITLE
Introduced new Slider component.

### LINKED ISSUE #192: Mobile Responsiveness for Sign-up Modal Avatar
**Current state**
The avatar images in the Sign-up Modal on mobile screen size appear in the following way:
Screenshot 2023-02-24 at 11 58 31 AM

**Expected state**
According to the [figma mockup](https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=2019%3A22335&t=0yhLFQ2wwqOYCPOi-0) the avatar images are required to appear in the following way:
Screenshot 2023-02-24 at 11 59 46 AM

### PR DESCRIPTION
**Pull request by shaavan**
Fixes #192

- This will allow displaying a horizontal slider when the window size is smaller than 767px. And otherwise, show the children in 5 column grid style.

- Used the Slider component in Signup.tsx file.

**Screenshot:**

_Introduced dummy avatars to show the difference._

|Desktop|Mobile|
|---------|------|
|Screenshot 2023-02-27 at 8 06 42 PM|Screenshot 2023-02-27 at 8 06 42 PM|

## Pull Request checklist

Please review the below PR requirements:

- [x] Build was run locally, and any changes were pushed
- [x] Ensure that the title clearly describes the changes
- [x] Issue to be referenced in the PR description rather than in the commits or PR title
- [x] Clean commit history

### CODE REVIEW

**Comment by afterburn** [comment]
image

@GBKS Do you think the padding of the parent element cutting off the avatars here is acceptable or would you rather we implement a solution that makes sure the avatars only get cut off by the actual screen borders?

File: components/Modals/SignUp.tsx [hunk 1]
```diff
@@ -58,21 +59,20 @@ export default function SignUpModal({ open, onClose, onConfirm }) {
 
         <h2 className="mb-4 text-xl font-bold">Choose an avatar</h2>
 
-        <div className="mb-5 flex cursor-pointer flex-wrap justify-between">
+        <Slider>
```
Comment (afterburn) [inline components/Modals/SignUp.tsx:62]: Let's rename this to ` HorizontalScrollView` or something akin to that. I'd reserve Slider for [these](https://www.w3schools.com/howto/howto_js_rangeslider.asp).

File: components/Slider.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export default function Slider({ children }) {
+  return (
+    <>
+      <div
+        className="scroll mb-2 flex cursor-pointer
```
Comment (afterburn) [inline components/Slider.tsx:5]: Nice, I wasn't sure if this would be possible without JS. Have you tested this on a variety of devices?

**Review by afterburn — CHANGES_REQUESTED** [review]
Nice work, I left one small comment

File: components/Slider.tsx (hunk 2 above)
Comment (shaavan) [reply components/Slider.tsx:5]: Thanks!

I have a Mac M1 pro (MacOS 13.1) and iPhone 13 (iOS 16.2). I tested this PR on both devices, and it runs perfectly.

However, we might encounter a problem with the scroll bar on the slider with Android.

On iOS (Safari and Chrome), the scrollbar automatically hides when the user is not scrolling. But I am unsure if this behavior will persist in Android and other OSes.

**Comment by shaavan** [comment]
Updated from 9716d5f17b121c9ceff7080f1ac160cd5c97e3ee to 339d190d6fe8f5a054e46f8e988d834c63f40c4e ([pr204.01](https://github.com/shaavan/saving-satoshi/commits/pr204.01) -> [pr204.02](https://github.com/shaavan/saving-satoshi/commits/pr204.02), [diff](https://github.com/shaavan/saving-satoshi/compare/pr204.01..pr204.02))
Addressed @afterburn comment

Changes:

1. Rename `Slider` -> `HorizontalScrollView`.

File: components/Modals/SignUp.tsx (hunk 1 above)
Comment (shaavan) [reply components/Modals/SignUp.tsx]: Sure!

Addressed in 339d190d6fe8f5a054e46f8e988d834c63f40c4e

File: components/Slider.tsx (hunk 2 above)
Comment (afterburn) [reply components/Slider.tsx]: Seems to work fine on Android, scrollbar is visible and hides itself after a timeout:

image

Let's get this merged and fix any issues in a separate PR if they are reported.

**Review by afterburn — APPROVED** [review]

**Comment by afterburn** [comment]
@ecurrencyhodler feel free to merge this after your tACK 😁

**Pull request merged**